cep-XXXX.md

```diff
@@ -0,0 +1,561 @@
+
+### Early Vision (2012-2014)
+
+The conda project's earliest discussions reveal a consistent vision for broad package manager interoperability that extends beyond just PyPI integration:
+
+- Issue [#307](https://github.com/conda/conda/issue/307) (2013) proposed conda should work with pip, npm, gem, rpm, and brew, envisioning conda as a universal package manager interface
+- Issue [#292](https://github.com/conda/conda/issue/292) (2013) explicitly titled "Making conda the ultimate package manager" outlined ambitious cross-ecosystem integration
+- Issue [#224](https://github.com/conda/conda/issue/224) (2012) discussed deprecating pip commands in favor of native conda functionality
+
+### The PyPI Integration Debate (2013-2016)
+
+Two competing philosophies emerged on how or if we should provide better integration with PyPI.
+
+#### Pro-integration
+
+Issue [#262](https://github.com/conda/conda/issue/262) (2013): conda should install directly from PyPI to reduce duplication and improve package availability.
+
+Arguments for this approach:
+
+- Users frequently need packages not available on conda channels
+- Building conda packages for every PyPI package is unsustainable
+
+#### Status quo
+
+Arguments to for maintaining an independent packaging ecosystem prevailed at the time for the following reasons:
+
+- Conda's value proposition is reproducible, binary-focused environments with precise dependency resolution
+- PyPI's source distributions and pip's resolver could compromise conda's guarantees
+- The conda-forge community successfully scaled recipe creation
+
+### Wheel Support Proposal (2017)
+
+Issue [#5202](https://github.com/conda/conda/issue/5202) proposed direct wheel installation support, recognizing:
+
+- Wheels provide binary distributions similar to conda packages
+- Growing wheel availability on PyPI reduced build complexity
+- Could bridge the gap between conda's reliability and PyPI's breadth
+
+### Broader Context (2018)
+
+The Python Discourse thread on [packaging scope boundaries](https://discuss.python.org/t/drawing-a-line-to-the-scope-of-python-packaging/883) reflects the larger ecosystem's struggle with:
+
+- Multiple competing package management tools (pip, conda, poetry, pipenv)
+- Unclear responsibilities and interoperability expectations
+- Need for clearer standards and communication between tools
+
+### conda-pypi development starts (2022)
+
+Jaime Rodríguez-Guerra starts development on the conda-pypi plugin aimed to improve conda and PyPI interoperability.
+
+### Rip development starts (2023)
+
+Prefix.dev starts a barebones pip implemented in Rust to resolve and install PyPI dependencies with Pixi.
+
+### Pixi Integrates with uv (Jan 2024)
+
+Pixi changes course to use uv directly instead of rip, which unlocks features like editable installations, and git and path dependencies.
```

Comment:
these are all now available for conda-only workflows through pixi-build.

Reply:
Hey @pavelzw, thanks so much for the feedback! Do you think we are missing a milestone in our brief history section? This pixi build feature is more about building path/git for conda packages than installing wheels isn't it?